extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -195,6 +190,20 @@ public void handle(AsyncResult<OAuth2Auth> event) {
         return new TenantConfigContext(auth, oidcConfig);
     }
 
+    @SuppressWarnings("deprecation")
+    private TenantConfigContext createdTenantContextFromPrivateKey(OAuth2ClientOptions options, OidcTenantConfig oidcConfig) {
```

Comment:
Maybe it should be `createdTenantContextFromPublicKey` instead ?

Reply:
@pedroigor ouch :-), will fix